internal/cmd/cmdutils.go

```diff
@@ -0,0 +1,32 @@
+package cmd
```

Comment:
we already have utils package, maybe this file can be moved there?

Reply:
[Like Dave Cheney](https://dave.cheney.net/2019/01/08/avoid-package-names-like-base-util-or-common), I think the package names should be as precise as possible. `utils` might contain anything - `cmd` will contain CLI-related stuff.